pwa/src/templates/components/ComponentsTemplate.module.css

```diff
@@ -32,12 +32,14 @@
 
 .filtersAndResultsContainer {
   display: flex;
+  flex-direction: column;
   gap: var(--web-app-size-lg);
 }
 
 .verticalFilters {
   flex: 1;
   min-width: 280px;
+  height: min-content;
```

Comment:
I don't think this is actually doing anything.

Reply:
this makes sure the grey background follows the filter collapsible when it collapses.
otherwise it would end up like this:
![image](https://user-images.githubusercontent.com/30209145/199454049-2ffaf8e1-a14d-4018-a4c9-e6d7c29120fc.png)